claims.json

```diff
@@ -856,6 +856,8 @@
   "com.quadas konfig*": "scalacenter/scaladex-void",
   "com.quartethealth datomisca*": "scalacenter/scaladex-void",
   "com.queirozf scala-date-time-utils*": "scalacenter/scaladex-void",
+  "com.rallyhealth play-json*": "rallyhealth/play-json-ops",
+  "com.rallyhealth scalacheck*": "rallyhealth/scalacheck-ops",
```

Comment:
This is now automatically being indexed: https://index.scala-lang.org/rallyhealth/scalacheck-ops